qa/tasks/cephfs/test_failover.py

```diff
@@ -309,6 +310,24 @@ def test_thrash(self):
 
         self.fs.wait_for_daemons(timeout=90)
 
+class TestFailoverBeaconHealth(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 2
+
+    def test_replay_beacon_estimated_time(self):
+        """
+        That beacon emits warning message with estimated time to complete replay
+        """
+        self.config_set("mds", "mds_tick_interval", "1")
```

Comment:
Why?

Reply:
I was expecting a sleep so you could artificially lengthen replay:

https://github.com/ceph/ceph/pull/52527#pullrequestreview-1537018461

This is not just so you can see the message without raciness but also so you can verify the estimate.